azure-mgmt-samples/pom.xml

```diff
@@ -68,6 +68,11 @@
         </dependency>
         <dependency>
             <groupId>${project.groupId}</groupId>
+            <artifactId>azure-core-test</artifactId>
```

Comment:
It was reported by Raj Vengalil during the hackfest that it was needed to
build / work on IntelliJ.

On Fri, Aug 21, 2015 at 9:46 AM, Jianghao Lu <notifications@github.com>
wrote:

> In azure-mgmt-samples/pom.xml
> <https://github.com/Azure/azure-sdk-for-java/pull/502#discussion_r37652990>
> :
>
> > @@ -68,6 +68,11 @@
> >          </dependency>
> >          <dependency>
> >              <groupId>${project.groupId}</groupId>
> > +            <artifactId>azure-core-test</artifactId>
>
> Why is this needed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Azure/azure-sdk-for-java/pull/502/files#r37652990>.
>